eng/tools/TypeSpecValidation/package.json

```
@@ -0,0 +1,15 @@
+{
+    "name": "typespec-validation",
+    "version": "0.0.1",
+    "type": "module",
+    "main": "dist/TypeSpecValidation.js",
+    "bin": {
+        "tsv": "cmd/tsv.js"
+    },
+    "dependencies": {
+        "simple-git": "^3.16.0"
+    },
+    "devDependencies": {
+        "@types/node": "^20.3.0"
```

```suggestion
        "@types/node": "^16.18.36"
```

I believe the version of this package is supposed to align with the minimum version of Node we want to support (`16` in our case).